README.md

```diff
@@ -106,6 +106,6 @@ Please go through [CSI Spec](https://github.com/container-storage-interface/spec
 Dependencies are managed through go module. To build the project, first turn on go mod using `export GO111MODULE=on`, then build the project using: `make`
 
 ### Testing
-* To create binary, run: `make bin/ibm-powervs-block-csi-driver`
+* To create binary, run: `make bin/powervs-csi-driver`
```

Comment:
This change is not needed now as the makefile changed.
Change: https://github.com/ppc64le-cloud/ibm-powervs-block-csi-driver/blob/51e4e2249f0d5eac4b418c9a4976a63678304e34/Makefile#L31